Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -170,24 +141,63 @@ extension FeatureFormExampleView {
     /// A Boolean value indicating whether general form workflow errors are presented.
     private var alertIsPresented: Binding<Bool> {
         Binding {
-            self.alertError != nil
+            submissionError != nil
         } set: { newAlertIsPresented in
             if !newAlertIsPresented {
-                self.alertError = nil
+                submissionError = nil
             }
         }
     }
     
+    /// The feature form view shown in the sheet over the map.
+    private var featureFormView: some View {
+        FeatureFormView(root: featureForm!, isPresented: featureFormViewIsPresented)
+            .onFormEditingEvent { editingEvent in
+                if case .savedEdits = editingEvent,
+                   let table = featureForm?.feature.table as? ServiceFeatureTable,
+                   !editedTables.contains(where: { $0 === table }) {
+                    editedTables.append(table)
+                }
+            }
+    }
+    
     /// A Boolean value indicating whether the form is presented.
     private var featureFormViewIsPresented: Binding<Bool> {
```

Comment:
This too should just be state.

Reply:
Done in [fe487f6](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/1220/commits/fe487f647cd804c0cc680a9e910709543e99aa98)